README.md

```diff
@@ -1 +1,22 @@
-# framegrabber
+FRAMEGRABBER
+============
+Driver module to load video frames captured from ARP cameras in to memory.
+
+Maintained at https://github.com/pelagicore/framegrabber
+
```

Comment:
This should be in a separate file called LICENSING or something similar imho?

Reply:
Perhaps, but usually you have a license in LICENSE or COPYING and you explain contribution policy in CONTRIBUTING and/or README. But naturally this is not set in stone.

Reply:
Exactly, the license should be split into a separate file was my point. I forgot the exact name of it when I was commenting before.

Reply:
The LICENSE file has already been delivered by Jeremiah in another commit.